include/wlr/types/wlr_output_management_v1.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * This an unstable interface of wlroots. No guarantees are made regarding the
+ * future consistency of this API.
+ */
+#ifndef WLR_USE_UNSTABLE
+#error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
+#endif
+
+#ifndef WLR_TYPES_WLR_OUTPUT_MANAGEMENT_V1_H
+#define WLR_TYPES_WLR_OUTPUT_MANAGEMENT_V1_H
+
+#include <stdbool.h>
+#include <wayland-server.h>
+#include <wlr/types/wlr_output.h>
+
+struct wlr_output_manager_v1 {
+	struct wl_display *display;
+	struct wl_global *global;
+	struct wl_list resources; // wl_resource_get_link
+
+	struct wl_list heads; // wlr_output_head_v1::link
+	uint32_t serial;
+
+	struct {
+		struct wl_signal apply; // wlr_output_configuration_v1
+		struct wl_signal test; // wlr_output_configuration_v1
+		struct wl_signal destroy;
+	} events;
+
+	struct wl_listener display_destroy;
+
+	void *data;
+};
+
+struct wlr_output_head_v1_state {
+	struct wlr_output *output;
+
+	bool enabled;
+	struct wlr_output_mode *mode;
+	struct {
+		int width, height;
+		int refresh;
+	} custom_mode;
+	int32_t x, y;
```

Comment:
Yes, that's right. It might be something we'll want to change in a future protocol major version. That said, if s is the scale, s pixels will physically have the size (in mm) of one normal (scale=1) pixel. Not sure it's worth it basically.